src/AzureAuth/CommandMain.cs

```diff
@@ -347,9 +347,34 @@ public int OnExecute()
             // Small bug in Lasso - Add does not accept a null IEnumerable here.
             this.eventData.Add("settings_scopes", this.authSettings.Scopes ?? new List<string>());
 
+            if (this.InteractivityDisabled())
+            {
+                this.logger.LogCritical($"Skipping user based authentication as specified by the environment variable : {EnvVars.DisableInteractivity}.");

```

Comment:
Strictly speaking this might not be true if the alternative environment variable is being respected. You could just not display any environment variable names and write
```suggestion
                this.logger.LogCritical("Interactive authentication is disabled.");

```